lib/graph/transpile.js

```diff
@@ -50,6 +50,8 @@ var transpileNode = function(node, outputFormat, options, graph, loader){
 			opts.transpiler = loader.transpiler || "babel"; // Babel is the default in Steal 1.x
 			if(loader.babelOptions) {
 				opts.babelOptions = loader.babelOptions;
+				opts.babelOptions.customPlugins =
```

Comment:
@matthewp I ended up adding an extra property to babel options with an object that map plugin names to their functions.

Then [`transpile`](https://github.com/stealjs/transpile/pull/66/files#diff-915cea4f628f9a2bb31bde126f493069R17) only cares about registering the plugins in this object.

The thing I don't like is adding an undocumented property to the babelOptions object, wondering if we should obfuscate the name somehow...